### DISCUSSION TITLE
How can one develop a user registration system with PHP OOP and Smarty template engine.

### DISCUSSION THREAD

**Discussion by arize99** [smarty-php/smarty#1033] [General]
Hello guys, I can't seem to find any pointers on how to achieve this task. The proctor specifically asked for this. I can make a simple user registration system with MySQL and basic PHP. I am not proficient with using engines, especially Smarty, I do not see any tips anywhere.
Any help?

Thank you.

**Comment by scottchiefbaker** [comment]
This is not really a Smarty question, as it's really more of a raw PHP / MySQL question. A quick Google search found this:

https://codewithawa.com/posts/complete-user-registration-system-using-php-and-mysql-database

Just search for `php user registration` and you'll find lots of examples.

**Reply by arize99** [reply]
Thank you @scottchiefbaker for the quick response. If I understand you correctly I just need to build a simple user registration system and ignore that they mentioned Smarty. ?

**Reply by scottchiefbaker** [reply]
Yes exactly. Smarty is great, but for this you want to figure out the basics using raw PHP.

If you want to make it fancier later you can come back to Smarty.

**Reply by arize99** [reply]
In this case, I want to use Smarty, I don't think they want raw PHP.
